rubocop.yml

```diff
@@ -400,17 +400,17 @@ Style/AlignParameters:
                  Align the parameters of a method call if they span more
                  than one line.
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-double-indent'
-  Enabled: false
+  Enabled: true
 
 Style/AndOr:
   Description: 'Use &&/|| instead of and/or.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-and-or-or'
-  Enabled: false
+  Enabled: true
```

Comment:
This is a problem in some cases I recall where you have to use `or` when raising an exception as part of the expression. Example: https://github.com/ForwardFinancing/ach_client/blob/master/lib/ach_client/objects/return_codes.rb#L30